tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -34,6 +34,13 @@ DEBUG_START
 #              |
 #              |
 # PCM3 <---- demux  <----- ALH0x103 (Speaker -max98373)
+ifelse(PLATFORM, `adl', `
+# PCM2 ----> smart_amp ----> ALH0x202 (Speaker -max98373)
+#               ^
+#               |
+#               |
+# PCM3 <---- demux  <----- ALH0x203 (Speaker -max98373)
+', `')
```

Comment:
I'd remove the ifelse section and just have something generic in the document like:
```
# PCM2 ----> smart_amp ----> ALH0xy02 (Speaker -max98373)
#              ^
#              |
#              |
# PCM3 <---- demux  <----- ALH0xy03 (Speaker -max98373)
```

Reply:
Ok sure, sound good! . Thanks @kv2019i